backend/accessibility_templates/aa-001.xml

```diff
@@ -0,0 +1,9 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->
```

Comment:
The coding format should be DVB DASH and HbbTV compliant, in this case EBU-TT-D in the version EBU-TT-D 1.0.1 with the termin ID 3.2.2 As EBU-TT-D is also conforms to IMSC a second CodingFormat element should be added with the termID 3.6.2.2 for IMSC 1.0.1.

Reply:
@paulhiggs Yes, if subtitles are provided with an MPEG DASH Live Stream. Is there a way to review the generated service list, that will be generated for the client application?

Reply:
there are a couple of service lists that include AccessibilityAttributes in the [servicelists](https://github.com/DVBProject/DVB-I-Reference-Client/tree/master/backend/servicelists) directory.
I initially updated the `[TM-STREAM0075r1 DVB-DASH_Reference_Streams.xml](https://github.com/DVBProject/DVB-I-Reference-Client/blob/master/backend/servicelists/TM-STREAM0075r1%20DVB-DASH_Reference_Streams.xml)` service list based on what is provided in the relevant DASH manifests.